packages/analytics-js/__tests__/components/capabilitiesManager/detection/dom.test.ts

```diff
@@ -8,6 +8,7 @@ describe('Capabilities Detection - DOM', () => {
     expect(isDatasetAvailable()).toBeTruthy();
   });
   it('should detect not supported javascript features', () => {
+    navigator.sendBeacon = jest.fn();
```

Comment:
you can configure this one globaly too if you like for all unit tests

Reply:
Done